### ISSUE TITLE
ehl_crb: tests/arch/x86/info failed.

### ISSUE THREAD

**Issue by chen-png** [zephyrproject-rtos/zephyr#33545]
**Description**
nothing output.

**To Reproduce**
1. west build -b ehl_crb -p auto tests/arch/x86/info
2. copy binary to USB/PXE boot media
3. boot

**Logs and console output**
nothing from console output

**Expected**
should see testcases results from console.

**Environment (please complete the following information):**
 - OS: Fedora33
 - Toolchain: zephyr-sdk-0.12.3
 - Commit ID: 911ba334a06a

**Comment by jenmwms** [comment]
I cannot reproduce this issue. I tested at the reported commit revision 911ba33 and again at more recent revision a9b4cae0 in case it has been resolved. Both configurations Pass for me using twister.

@chen-png @jhedberg Are you able to reproduce this reported issue?

**Comment by jenmwms** [comment]
Oh interesting, I do not see it with SDK 12.1 but I do see it with SDK 12.3. I'll keep testing and see what we find.

**Comment by jhedberg** [comment]
@jenmwms I haven't tried with twister, but manually building the info sample and booting the zephyr.efi through PXE (on `ehl06` in our lab, FWIW) gets stuck at this point:
```
*** Zephyr EFI Loader ***
Zeroing 4774384 bytes of memory at 0x106000
Copying 32768 data bytes to 0x1000 from image offset
Copying 24576 data bytes to 0x100000 from image offset 32768
Copying 6317568 data bytes to 0x593a00 from image offset 57344
Jumping to Entry Point: 0x1123 (48 31 c0 48 31 d2 48)
```

E.g. hello_world using the same build process and machine works fine:
```
Jumping to Entry Point: 0x10f7 (48 31 c0 48 31 d2 48)
*** Booting Zephyr OS build zephyr-v2.5.0-1589-g8965df96222e  ***
Hello World! ehl_crb
```

**Comment by jenmwms** [comment]
@jhedberg Thanks for the feedback! I will sort out my env issues asap.

**Comment by jenmwms** [comment]
I can reproduce the issue now, I'll work on bisect and debug to update here soon.

**Comment by jenmwms** [comment]
Noting that this is still an issue. This one is not simple to bisect based on last known working point in history, unknown when that was so moving forward with debug otherwise.

**Comment by jenmwms** [comment]
@chen-png Do you know if you this works on other 64-bit (up_squared)? 
FWIW - Was hoping to test against an emulator if possible, but this test allows qemu_x86 and not qemu_x86_64 so it fails. Shouldn't it work for qemu_x86_64 if it works for 64-bit boards like ehl_crb and up_squared?

**Comment by andyross** [comment]
Indeed, this hangs immediately after the boot banner (and before entering main()) on qemu_x86_64.  My guess is this is just bitrot with 64 bit support for multiboot or something.

**Comment by jenmwms** [comment]
Re-tested with that PR above - it resolves the issue.

**Comment by chen-png** [comment]
>Do you know if you this works on other 64-bit (up_squared)?

it works on up_squared.
